### ISSUE TITLE
[Profesiones] Herboristería

### ISSUE THREAD

**Issue by titoalfon** [RoDNet/Bugtracker-Infinium-4.3.4#2986]
Era ingeniero (completo) con especialización en ingeniería gnómica pero abandoné la profesión y empecé con herboristería, en su lugar. Hasta ahí todo bien.
Pero al salir de wow y borrar la caché  me encuentro con que en el libro de hechizos aparece Ingeniería y  aparece,a la derecha el icono de ingeniería gnómica,en lugar de la habilidad "Sangre de vida", de hecho esta habilidad tambien ha desaparecido de la barra de acción, quedando en blanco y no tampoco aparece el de ingeniería. No obstante puedo seguir recolectando hierbas y subir la habilidad aunque no me dan los títulos correspondientes sólo va subiendo la experiencia.
![image](https://f.cloud.github.com/assets/5804269/1746852/d9a6bd4c-644a-11e3-9ddb-e9e2ded52d24.png)

**Comment by ghost** [comment]
sí, cuando se aprende una profesión y una especialización y luego se olvida para aprender otra, te quedas sin profesión. Lleva reportado como 2 meses o así. https://github.com/RoDNet/Bugtracker-Infinium-4.3.4/issues/2437

A todos los que sufren el bug les dicen lo mismo, que vengan aquí a reportarlo. "Sólo" te queda esperar a que lo arreglen, no puedes hacer nada, has perdido una profesión hasta entonces lo siento jajaja

**Comment by kingtonix** [comment]
Bueno a unirme a la fiesta ya que llevo 2 meses esperando poniendolo en foro (Porque me dijeron los mjs),poniéndolo en el bugtraker (porque también me dijeron los mjs) haber si esta vez  es diferente JAJA #2936
